onnxruntime/core/framework/execution_frame.h

```diff
@@ -162,6 +173,8 @@ class ExecutionFrame final : public IExecutionFrame {
     return static_activation_memory_sizes_in_byte_;
   }
 
+  size_t program_counter_;
```

Comment:
>size_t program_counter_; [](start = 2, length = 24)

public member variable is not a good coding style..

Reply:
agreed.

---
In reply to: [598071358](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r598071358) [](ancestors = 598071358)

Reply:
good idea.

---
In reply to: [598864650](https://github.com/microsoft/onnxruntime/pull/7077#discussion_r598864650) [](ancestors = 598864650)